packages/relay/src/lib/services/ethService/ethCommonService/index.ts

```diff
@@ -141,7 +137,16 @@ export class CommonService implements ICommonService {
 
       if (fromBlockNum > toBlockNum) {
         return false;
-      } else if (toBlockNum - fromBlockNum > blockRangeLimit) {
+      }
+
+      const blockRangeLimit =
+        process.env.TEST === 'true'
```

Comment:
`process.env.TEST` what is this property doing?

Reply:
This was already used in `eth_getLogs`. As discussed, this was removed.